PastaGA.py

```diff
@@ -1,17 +1,92 @@
+__author__ = 'theep_000'
```

Comment:
Not following coding guidelines 

Know 'em, read 'em, love -em:

https://github.com/archerdxinc/docs/blob/master/developer/coding_guidelines.rst